frontend/src/setupTests.js

```diff
@@ -8,3 +8,15 @@ Enzyme.configure({ adapter: new Adapter() });
 global.requestAnimationFrame = (callback) => {
   setTimeout(callback, 0);
 };
+
+const location = JSON.stringify(window.location);
+delete window.location;
+
+Object.defineProperty(window, "location", {
+  value: JSON.parse(location),
```

Comment:
What's the purpose of stringifying the location and then parsing it again?

Reply:
I didn't spend too much time cleaning that up, but basically i believe that forced JS to create a new object with to override the old window.location with. and If you don't override it properly the following global.location override fails as well